tracer/test/Datadog.Trace.IntegrationTests/StatsTests.cs

```diff
@@ -227,7 +227,7 @@ public async Task SendsStatsWithProcessing_Obfuscator()
             await tracer.TracerManager.ShutdownAsync(); // Flushes and closes both traces and stats
 
             var statsPayload = agent.WaitForStats(1);
-            var spans = agent.WaitForSpans(13);
+            var spans = agent.WaitForSpans(6);
```

Comment:
Wow, that's a big change, do we have any idea when the number of spans here changed (i.e. _why_) or has this always been wrong do you think (certainly possible!)